sst/SimEngCoreWrapper.cc

```diff
@@ -277,8 +297,8 @@ void SimEngCoreWrapper::fabricateSimEngCore() {
   if (maxAddrMemory_ < coreInstance_->getProcessImageSize()) {
     output_.verbose(
         CALL_INFO, 1, 0,
-        "Error: SST backend memory is less than processImage size. Please "
-        "increase the memory allocated to memHierarchy.memBackend and "
+        "Error: SST backend memory is less than processImage size. "
+        "Please increase the memory allocated to memHierarchy.memBackend and "
```

Comment:
Again, does this print to stdout / console and need a prefix?

Reply:
This will already have the prefix as it its outputted using the `output_` class.